### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by lkasso**
- Add create_user_event() capability 
- Add 2Mhz PHY disable 
- Update miniz
- Fix anonymous dataloggers dataprocessor deletion
- Add steps URI for better logging identifiers
- Add additional comments
- Add additional test and comment tests

### CODE REVIEW

**Comment by lkasso** [comment]
Added an optional value to the spoof button event call

**Review by importRyan — APPROVED** [review]
Big thanks! I've tested this with the Combine SDK. Anonymous loggers no longer crashes, steps logger has a name, and button spoofing works.

File: src/metawear/core/debug.h [hunk 1]
```diff
@@ -79,6 +79,13 @@ METAWEAR_API void mbl_mw_debug_read_schedule_queue_usage(const MblMwMetaWearBoar
  */
 METAWEAR_API void mbl_mw_debug_spoof_notification(const MblMwMetaWearBoard *board, const uint8_t *value, uint8_t length);
 
+/**
+ * Creates a fake button event with the data 0x02.
+ * Requires that the switch signal be streaming or logging to add a spoofed event in the logger.
+ * @param board     Calling object
+ */
+METAWEAR_API void mbl_mw_debug_spoof_button_event(const MblMwMetaWearBoard *board);
```
[the remark this answers is not in the document]
Comment (lkasso) [reply src/metawear/core/debug.h]: I made an update and added a UInt as something you can pass to the spoof_button_event command - please test this and let me know if it works ok.

[the remark this answers is not in the document]
Comment (importRyan) [reply src/metawear/core/debug.h]: Are you referring to [1afbc99](bhttps://github.com/mbientlab/MetaWear-SDK-Cpp/pull/41/commits/1afbc99ce6e521f3ebb13230de89601a0d9eb7be)? 

I've tested that and pushed it to the beta SDK main and approved this PR. Am I missing a commit?

[the remark this answers is not in the document]
Comment (lkasso) [reply src/metawear/core/debug.h]: Yes cc15be4

**Comment by importRyan** [comment]
Cool, yeah, it works! Super awesome.

**Review by importRyan — APPROVED** [review]

**Pull request merged**